cmd/yggd/main.go

```diff
@@ -140,6 +140,12 @@ func setupClient(
 		if err != nil {
 			return nil, nil, cli.Exit(fmt.Errorf("cannot create HTTP transport: %w", err), 1)
 		}
+	case "", "noop":
```

Comment:
I would use only "noop". I would not use "" as another option for this "protocol". I will explain why. 

Common practice is following. When "" is used for value of some option, then default option is used (default protocol is "mqtt"). Thus suggested behavior would be confusing. The `"noop"` could be used in `/etc/yggdrasil/config.toml` for option `protocol`, right?

BTW: Is the "noop" the best keyword?  I think that it is not so well known word. I tried to find it in some dictionaries and most dictionaries does not contain this word. What about "none" keyword for this purpose? If the "noop" was used, then I would add some description to man page.  The problem is that we generate man page.  Adding additional information to man page would probably require adding custom function for generating man page.

Reply:
Hm. Yes. I thought about whether I should include "" or not. You make a good point. I'll drop it from the case condition. I went around and around on a name, including `LoopTransport`, `NullTransport` and a few others. Neither name really fit the functionality of the transport until I settled on the term "no-op". It's an abbreviation for "no operation". I felt this name fit the functionality of the transport quite well, as it doesn't loop anything back; it just takes no operation on the data. I like the term "none" to describe the protocol. Any objections against?

```suggestion
	case "none", "noop":
```